cas/src/main/java/org/springframework/security/cas/web/CasAuthenticationFilter.java

```diff
@@ -242,7 +242,7 @@ protected final void successfulAuthentication(HttpServletRequest request,
 	@Override
 	public Authentication attemptAuthentication(final HttpServletRequest request,
 			final HttpServletResponse response) throws AuthenticationException,
-			IOException {
+			IOException, ServletException {
```

Comment:
Please do not make changes unless they are necessary for this PR. Please go through the PR and remove unnecessary changes.

Reply:
fixed